pkg/util/provider/metrics/metrics.go

```diff
@@ -80,6 +80,20 @@ var (
 		Help:      "Number of Failed Cloud Service API requests, partitioned by provider, and service.",
 	}, []string{"provider", "service"},
 	)
+
+	APIRequestDuration = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: namespace,
+		Subsystem: cloudAPISubsystem,
+		Name:      "api_request_duration",
```

Comment:
```suggestion
		Name:      "api_request_duration_seconds",
```
duration metrics usually have the time unit in their name